docs/reference/release-notes/7.14.asciidoc

```diff
@@ -1,3 +1,34 @@
+[[release-notes-7.14.2]]
+== {es} version 7.14.2
+
```

Comment:
```suggestion
coming::[7.14.2]
```

Reply:
@henningandersen, it's not required to add this tag, but would allow us to merge before release day. I can then create a subsequent PR to remove that tag on release day. This is my first time serving as the docs release coordinator, so I'm trying to follow conventions already in place. This change isn't mandatory though!